distributed/shuffle/_scheduler_plugin.py

```diff
@@ -119,6 +119,9 @@ def get_or_create(
         key: str,
         worker: str,
     ) -> ToPickle[ShuffleRunSpec]:
+        # FIXME: Sometimes, this doesn't actually get pickled
```

Comment:
which times are sometimes? Can create an issue about this and link it?

Reply:
My suspicion is that it's related to `inproc://`, but I've also seen this mentioned in other parts of the codebase, so I'm not sure about that. I was planning to investigate this soon(ish), but there are more pressing issues for now.